### PR TITLE
Do not round things in stripe

### PR DESCRIPTION
**Pull request by marcoambrosini**
Signed-off-by: marco <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 1]
```diff
@@ -508,7 +508,8 @@ export default {
 		},
 
 		showQualityWarningTooltip() {
-			return this.qualityWarningTooltip && (!this.isQualityWarningTooltipDismissed || this.mouseover)
+			return true
+			// return this.qualityWarningTooltip && (!this.isQualityWarningTooltipDismissed || this.mouseover)
```
Comment (nickvergessen) [inline src/components/CallView/shared/LocalMediaControls.vue:512]: ???

Comment (marcoambrosini) [reply src/components/CallView/shared/LocalMediaControls.vue]: you should have waited 10 seconds!

**Comment by nickvergessen** [comment]
Counter proposal in https://github.com/nextcloud/spreed/pull/6521

**Comment by marcoambrosini** [comment]
> Counter proposal in #6521

continuing there

**Pull request closed without merging**